stringassignment.js

```diff
@@ -0,0 +1,42 @@
+// 1. `console.log` the first character of a string.
```

Comment:
I see some instances of hard coding which is a practice to be avoided. Your code must be able to accommodate for different inputs and still returns consistent and correct outputs. Try to revisit some questions again and come ask me and Alejo and questions. Also try to do bonus questions next time as they are really good practice to advance your mastery. Keep practicing, keep coding and keep up the good work!